CONTRIBUTING.md

```diff
@@ -52,6 +52,9 @@ MAVEN_OPTS='--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/ja
 
 ### Running the Yarn frontend build
 
+> [!TIP]
+> If you already have Node.js installed, you do not need to change your path. Start using `yarn` by enabling [Corepack](https://yarnpkg.com/corepack) `corepack enable`, if it isn't already; this will add the yarn binary to your PATH.
```

Comment:
> Start using `yarn` by enabling [Corepack](https://yarnpkg.com/corepack) `corepack enable`

Should there be a preposition before `corepack enable`?

> Start using `yarn` […]; this will add the yarn binary […].

Why is the reference to Yarn in monospaced text at the beginning of the sentence but not at the end of the sentence?

Reply:
`yarn`  is a command which is lowercase. ...but please use the suggest function in github

Reply:
I am not a native speaker and the aove is adopted from https://yarnpkg.com/getting-started/install

Reply:
> the above is adopted from https://yarnpkg.com/getting-started/install

For reference, the original material reads:

> Start by enabling [Corepack](https://yarnpkg.com/corepack), if it isn't already; this will add the `yarn` binary to your PATH:
> `corepack enable`

The original material correctly uses a colon to separate `corepack enable` from the beginning of the sentence. In contrast, the material in this PR does not separate `corepack enable` (which functions as a noun) from the sentence in any way:

> Start using `yarn` by enabling [Corepack](https://yarnpkg.com/corepack) `corepack enable`

Here Corepack is the object of "enabling", and `corepack enable` (functioning as a noun) is left dangling.

One way to correctly use `corepack enable` as a noun in the middle of the sentence would be as an object of the preposition "with" (e.g., "with `corepack enable`").

Reply:
Thanks for explaining!